signUp.html

```diff
@@ -0,0 +1,83 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>Sign Up</title>
+<style type="text/css">
+body {
+	background: url("http://wp.givingtuesday.ca/app/uploads/2014/11/GivingTuesday_BlogHeader.jpg") no-repeat center center fixed;
```

Comment:
* Why not a separate CSS file?
* Better to download the image and serve as a local resource than serving over the internet.